src/CheckOut.js

```diff
@@ -280,7 +280,10 @@ class CheckOut extends React.Component {
   async onSessionEnd() {
     const {
       resources: { activeRecord: { patronId } },
-      mutator: { endSession: { POST: endSession } },
+      mutator: {
+        endSession: { POST: endSession },
+        activeRecord: { update: updateParonId }
```

Comment:
As long as you're in here cleaning things up, can you fix the typo in this method name? Please use `updatePatronId`.

Actually: where is this method defined? I see the `propTypes` definition for `mutator.activeRecord.update` but no reference to `updateParonId` (or the correct `updatePatronId`) anywhere in this repo 😕

Reply:
I think this is reassignment in js notation. The real name `update` was named to be `updateParonId ` for more context. So typo is only here and needs to be handled.